appengine/standard-cloud-tasks/Gemfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+gem "google-cloud-tasks", "2.1.1"
```

Comment:
Use `"~> 2.1"` (instead of `"2.1.1"`) for the version restriction. (And then please regenerate the corresponding Gemfile.lock once you've also modified the sinatra line below.)

Reply:
done

Reply:
It still looks like it's `"= 2.1.1"`.

Reply:
Sorry, thought you were still talking about the version for sinatra.

Reply:
Done.